android/src/main/java/com/rnldk/RnLdkModule.kt

```diff
@@ -604,6 +632,14 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
       // we don't route as we are a light mobile node
     }
 
+    if (event is Event.PaymentClaimed) {
+      // do we even want an event for a successfull incoming ln payment..?
```

Comment:
so, if i got it right event `Event.PaymentReceived` is triggered before we even claim, and  `PaymentClaimed` is triggered when we successfully claim?

can we try to claim and fail? otherwise its kinda all the same to me, and to upper level code which just wants to be paid

Reply:
@TheBlueMatt ^^^

Reply:
@TheBlueMatt fixed, can you pls take a look?